packages/prover/src/utils/consensus.ts

```diff
@@ -13,7 +13,7 @@ export async function fetchNearestBlock(
 ): Promise<capella.SignedBeaconBlock> {
   const res = await api.beacon.getBlockV2(slot);
 
-  if (res.ok) return res.response.data;
+  if (res.ok) return res.response.data as capella.SignedBeaconBlock;
```

Comment:
The new typescript found this type issue?

Reply:
yes the newer version is strict when returning union type.